draft-song-lake-ra.md

```diff
@@ -476,16 +328,228 @@ Request_structure = {
 }
 ~~~~~~~~~~~~~~~~
 
-### Result {#result}
+##### Result {#result}
 
 The attestation result is generated and signed by the Verifier as a serialized EAT {{I-D.ietf-rats-eat}}.
 The Relying Party can decide what action to take with regards to the Attester based on the information elements in attetation result.
 
 The EAD item is:
 
-* ead_label = TBD6
+* ead_label = TBD3
 * ead_value is a serialized EAT.
 
+##### trigger_pp {#trigger-pp}
+
+The EAD item trigger_pp is used when the sender triggers the receiver to start a remote attestation in passport model.
+The receiver MUST reply with an EAD item for passport model.
+The ead_value can be empty, using ead_label plays the role of the trigger.
+
+The EAD item is:
+
+* ead_label = TBD4
+* ead_value = null
+
+## EDHOC Message Flow {#flow}
+
+The EDHOC protocol defines two types of athentication message flows.
+In this specification, we support both the EDHOC remote message flow and the EDHOC reverse message flow (see {{Appendix A.2.2 of RFC9528}}) to perform remote attestation.
```

Comment:
The EDHOC protocol defines two possible message flows, namely the EDHOC forward and reverse message flows (see {{Appendix A.2.2 of RFC9528}}).
They establish whether the EDHOC initiator acts as a CoAP client or server (and vice-versa for the responder).
In this specification, both flows can be used to perform remote attestation.